src/rtc/h5_prep.py

```diff
@@ -795,7 +795,7 @@ def get_metadata_dict(product_id: str,
              '_output_backscatter_expression_convention',
              ALL_PRODUCTS,
              'linear backscatter intensity',
-             'Backscatter expression convension'],
+             'Backscatter expression convention'],
```

Comment:
`convention` has been used throughout the code and the metadata field name, I think it's better to stick with that. 
Another similar issue is `normalization` vs. `normalisation`. In this case, I've left it as it is because it's what is used in CEOS webpage.

Reply:
Thank you, @seongsujeong , I had that change in #64 too =)